src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -138,6 +138,24 @@ namespace jhs = ns_smart_device_link::ns_json_handler::strings;
 
 using namespace ns_smart_device_link::ns_smart_objects;
 
+hmi_apis::Common_ServiceType::eType GetHMIServiceType(
+    protocol_handler::ServiceType service_type) {
```

Comment:
@mked-luxoft I can see ServiceType is part of PH. PH depends from API, so I suppose that this funciton can be moved to PH

Reply:
@LuxoftAKutsan  fixed in 007c20e6953023dc50553d14a4eaa354ff325fc0. One note though. I don't think that moving declaration of this function to `protocol_handler.h` is a good idea, since we need to include `HMI_API.h` and as a result add include path to it for every translation unit that includes `protocol_handler.h`. Hence, i moved definition of this function to `service_status_update_handler.cc`